modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/OrchestratorService.java

```diff
@@ -33,15 +37,23 @@
 @Component
 public class OrchestratorService implements ApplicationListener<ApplicationEvent> {
 
+    private static final int STATUS_MSG_MAX_LENGTH = 255;
+
+    private static final int CPU_COUNT = Runtime.getRuntime().availableProcessors();
+
+    public static ThreadPoolExecutor executorService = new ThreadPoolExecutor(CPU_COUNT * 2,
```

Comment:
Would it be possible to use the Spring Async framework instead of directly creating threads?

Reply:
Ok.